Sources/Extensions/UIKit/UIImageExtensions.swift

```diff
@@ -189,16 +189,16 @@ public extension UIImage {
     ///   - blendMode: how to blend the tint
     /// - Returns: UIImage tinted with given color.
     public func tint(_ color: UIColor, blendMode: CGBlendMode) -> UIImage {
-        let drawRect = CGRect(x: 0.0, y: 0.0, width: size.width, height: size.height)
+        let drawRect = CGRect(origin: .zero, size: CGSize(width: size.width, height: size.height))
         UIGraphicsBeginImageContextWithOptions(size, false, scale)
+        defer {
+            UIGraphicsEndImageContext()
```

Comment:
Why `defer` instead of using the original method? There's nowhere that this function is being exited early.

Reply:
Is just to avoid the have to store the variable to end the context and then return it. Just return the value and end the context on defer.